### PR TITLE
feat(blog): add form-js@1.6.0 blog post

### PR DESCRIPTION
**Pull request by vsgoulart**
Closes https://github.com/bpmn-io/bpmn.io/issues/185.

### CODE REVIEW

**Comment by vsgoulart** [comment]
@Skaiir  @nikku I can't add reviewers because I'm not a maintainer

**Comment by nikku** [comment]
@vsgoulart You should be a maintainer now.

**Review by nikku — COMMENTED** [review]
Thanks for the contribution :tada:.

Find my review feedback attached:

* Screenshot "dynamic table", let's feature it, and ensure the following:
   * format the condition using multi-line, so that it is visible on the screen
   * If we don't use it, fix the spelling in the other one "static table", where it should be called `customer` or `consumer`, but not what is currently written there
* Let's be proud, but not brag. The title "unveiling dynamic XYZ" is super braggy, as if we'd save the world. Let's be concise, on point, regarding what we implement instead, i.e. "Dynamic Lists and Tables land in form-js".
* Table component:
  * I think it is worth highlighting that the data source can be static (directly taken from user input) or _computed_ from form context + input.
  * I believe the following _would work_: I define a list component, with inputs; I define a table view that sources the list component data _and_ displays a subset in the table view.

File: src/blog/2023/011-form-js-1.6.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+title: Unveiling Dynamic Forms
```
Comment (nikku) [inline src/blog/2023/011-form-js-1.6.md:2]:
```suggestion
title: Dynamic Lists and Tables land in form-js
```

File: src/blog/2023/011-form-js-1.6.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+description: "Explore what's new in form-js@1.6.0: Unveil the power of dynamic lists and an innovative table component, designed to enhance the functionality and versatility of your forms."
```
Comment (nikku) [inline src/blog/2023/011-form-js-1.6.md:3]:
```suggestion
description: "Explore what's new in forms: A powerful dynamic list and a new table component ship with form-js@1.6.0."
```

File: src/blog/2023/011-form-js-1.6.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+preview_image: http://bpmn.io/assets/attachments/blog/2023/011-dynamic-table.png
+layout: blogpost.hbs
+slug: 2023-form-js-1-6-0-unveiling-dynamic-forms
```
Comment (nikku) [inline src/blog/2023/011-form-js-1.6.md:6]:
```suggestion
slug: 2023-form-js-1-6-0-dynamic-list-table
```

File: src/blog/2023/011-form-js-1.6.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  - Vinicius Goulart <https://github.com/vsgoulart>
+  - Valentin Serra <https://github.com/skaiir>
+published: 2023-12-18 12:00
+
+releases:
+  - "form-js@1.6.0"
+---
+
+<p class="introduction">
+  We're thrilled to announce the [form-js@1.6.0 release](https://github.com/bpmn-io/form-js), a significant leap in form development. This version introduces dynamic lists and a new table component, elevating the capabilities of complex form creation.
```
Comment (nikku) [inline src/blog/2023/011-form-js-1.6.md:17]:
```suggestion
  We're thrilled to announce the [form-js@1.6.0 release](https://github.com/bpmn-io/form-js). It ships with dynamic lists and a new table component and unlocks new capabilities in complex form use-cases.
```

File: src/blog/2023/011-form-js-1.6.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+## Dynamic Lists
+
+Without manual work, dynamic lists enable the effortless generation of repetitive sets of form components, such as data tables or ordered lists.
+
+<div class="figure full-size">
+  <img src="{{ assets }}/attachments/blog/2023/011-dynamic-list.png">
+
+  <p class="caption">
+    Dynamic lists offer flexibility in modeling repetitive lists of form components.
```
Comment (nikku) [inline src/blog/2023/011-form-js-1.6.md:32]: Consider: Link this image to your demo (camunda forms).

Better: Get the bpmn.io form demo to feature the form playground, and link that.

File: src/blog/2023/011-form-js-1.6.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+The new table component in `form-js@1.6.0` brings an improved approach to data organization within forms, offering a structured and user-friendly interface. This addition simplifies the management and entry of complex data, simplifying overall form usability.
+
+<div class="figure full-size">
+  <img src="{{ assets }}/attachments/blog/2023/011-table-component.png">
+
+  <p class="caption">
+    Organize tabular data with the new table component, giving your users all the information they need to fill their forms in a space efficient way.
+  </p>
+</div>
+
```
Comment (nikku) [inline src/blog/2023/011-form-js-1.6.md:47]: Feature how the data source can be static (just input data) or computed from existing form data, i.e. a prior dynamic list.

File: src/blog/2023/011-form-js-1.6.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+## Wrapping Up
+
+Read through the [full changelog](https://github.com/bpmn-io/form-js/blob/master/packages/form-js/CHANGELOG.md#160) to learn about all issues that we addressed with `form-js@0.16.0`.
+
+Did we miss anything? Did you spot a bug, or would you like to suggest an improvement? Reach out to us via [our forums](https://forum.bpmn.io/), [follow us on Mastodon](https://fosstodon.org/@bpmn_io), or file an issue you found in [the form-js issue tracker](https://github.com/bpmn-io/form-js/issues).
+
+Get the latest Forms modeling toolkit pre-packaged or as source code via [npm](https://www.npmjs.com/package/@bpmn-io/form-js) or [unpkg](https://unpkg.com/@bpmn-io/form-js).
+
+---
```
Comment (nikku) [inline src/blog/2023/011-form-js-1.6.md:56]:
```suggestion
```

File: src/blog/2023/011-form-js-1.6.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+title: Unveiling Dynamic Forms
+description: "Explore what's new in form-js@1.6.0: Unveil the power of dynamic lists and an innovative table component, designed to enhance the functionality and versatility of your forms."
+preview_image: http://bpmn.io/assets/attachments/blog/2023/011-dynamic-table.png
```
Comment (nikku) [inline src/blog/2023/011-form-js-1.6.md:4]: :star:

**Comment by Skaiir** [comment]
Ah maaaaan :|

File: src/blog/2023/011-form-js-1.6.md (hunk 6 above)
Comment (nikku) [reply src/blog/2023/011-form-js-1.6.md:47]: Consider featuring how (also more complex) nested components can be stacked inside a dynamic list.

**Comment by vsgoulart** [comment]
@Skaiir you can close my PR in favor of yours since I'll be on vacation until January

**Pull request closed without merging**